src/modules/launcher/Wox.Infrastructure/Helper.cs

```diff
@@ -80,6 +80,7 @@ public static string Formatted<T>(this T t)
         }
 
         // Function to run as admin for context menu items
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "Exception is being logged")]
```

Comment:
@avneet-kr As discussed offline, we don't want to set a standard for ignoring this error by default.  Perhaps change the justification method to something like "Justification=Suppressing this to enable FxCop.  We are logging the exception, and going forward general exceptions should not be caught"